packages/flutter_tools/test/general.shard/features_test.dart

```diff
@@ -78,7 +78,7 @@ void main() {
     }));
 
     test('flutter web help string', () {
-      expect(flutterWebFeature.generateHelpMessage(), 'Enable or disable Flutter for web. This setting will take effect on the master and dev channels.');
+      expect(flutterWebFeature.generateHelpMessage(), 'Enable or disable Flutter for web. This setting will take effect on the master, dev, and beta channels.');
```

Comment:
nit: master, dev and beta

Reply:
I will defend the Oxford comma until my dying breath

Reply:
I agree with @jonahwilliams 

https://www.youtube.com/watch?v=W_WWx_ndVUA

Reply:
Nevermind I didn't know this is something almost mandatory in American English, lesson learned